### PR TITLE
console: rename IP whitelist to IP allowlist

### PR DESCRIPTION
**Pull request by ultraschuppi**
## Description

for https://github.com/camunda-cloud/camunda-cloud-management-apps/issues/1206

- [x]  screenshots are updated as well

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by christinaausley** [comment]
Also, relates to https://github.com/camunda/product-hub/issues/1360.

**Comment by ultraschuppi** [comment]
went to prod with went to prod with https://github.com/camunda-cloud/camunda-cloud-management-apps/releases/tag/8.4.22 - can be merged and released whenever convenient

File: docs/components/console/manage-clusters/manage-ip-whitelists.md
Comment (christinaausley) [inline docs/components/console/manage-clusters/manage-ip-whitelists.md:1]: I'm going to add a card to my backlog to adjust the id here and apply redirects, but fine with this fix for now 👍

**Review by christinaausley — DISMISSED** [review]

File: docs/components/console/manage-clusters/manage-ip-whitelists.md
Comment (christinaausley) [reply docs/components/console/manage-clusters/manage-ip-whitelists.md:1]: I also noticed we switch between uppercase and lowercase, which I'll add to this issue 👍

**Comment by christinaausley** [comment]
@ultraschuppi I backported a few instances I found -- let me know if you have any lingering comments. Otherwise, LGTM.

**Review by christinaausley — APPROVED** [review]

**Comment by ultraschuppi** [comment]
> @ultraschuppi I backported a few instances I found -- let me know if you have any lingering comments. Otherwise, LGTM.

i initially just wanted to do this on 8.4 and next - no strong opinions on my side for backporting to older versions as well

will merge now

**Pull request merged**